spec/order_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'order'
+
+describe Order do
+
+let(:order)  {Order.new}
+
+	it 'should have a list of dishes' do
+		expect(order.dishes_choice.length).to eq (4)
+	end
+
+	it 'should set the order items to 0' do
+		expect(order.order_count).to eq (0)
+	end
+
```

Comment:
order_count should have been tested for other numbers - you have missed edge cases. Always try to test the "good path" and the "bad path".